src/utils/ssmCommand.ts

```diff
@@ -8,26 +8,20 @@ const s3 = new AWS.S3();
 export function ssmCommand(command: string, instanceId: string, outputExpected: boolean = true): Promise<string> {
     return new Promise<string>((resolve, reject) => {
         sendCommand(command, instanceId)
-            .then((sendResult: SSM.Types.SendCommandResult) => wait(10000, sendResult.Command.CommandId))
-            .then((commandId: string) => getCommandResult(commandId, instanceId))
+            .then(delay)
+            .then((sendResult: SSM.Types.SendCommandResult) => awaitCommandResult(sendResult.Command.CommandId, instanceId, 3))
```

Comment:
Thinking about this some more, I would probably poll a few more times than 3. There's no real advantage to giving up early (particularly as AWS might have already submitted the command), so now we have polling functionality I think we can afford to be quite patient here.

Reply:
Good idea, we've bumped this up to 6 times, so a minute's worth of retries